src/mame/drivers/bbusters.cpp

```diff
@@ -192,64 +83,112 @@ If you calibrate the guns correctly the game runs as expected:
     in the order: Minimum axis 0, middle axis 0, maximum axis 0 (repeat for
     the other 5 axes).
 
+    ----------------------------------------------------------------------------------------
+
+    Beast Busters Region code works as follows
+
+    ROM[0x003954/2] = data * 4;
+
+    Country/Version :
+
+     - 0x0000 : Japan?
+     - 0x0004 : US?
+     - 0x0008 : World?    (default)
+     - 0x000c : World?    (same as 0x0008)
+
 ***************************************************************************/
 
 #include "emu.h"
-#include "includes/bbusters.h"
 
+#include "video/snk_bbusters_spr.h"
+#include "machine/gen_latch.h"
+#include "video/bufsprite.h"
+#include "screen.h"
+#include "tilemap.h"
 #include "cpu/z80/z80.h"
 #include "cpu/m68000/m68000.h"
 #include "machine/nvram.h"
 #include "machine/upd7004.h"
-#include "sound/ym2608.h"
 #include "sound/ym2610.h"
 #include "emupal.h"
 #include "screen.h"
 #include "speaker.h"
```

Comment:
You’ve duplicated `screen.h` now.  Please organise the headers from most dependent to least dependent (`src/mame`, then `src/devices`, then `src/emu`) and sort them alphabetically within sections.